packages/playwright/src/runner/testServer.ts

```diff
@@ -86,8 +90,7 @@ class TestServerDispatcher implements TestServerInterface {
           gracefullyProcessExitDoNotHang(0);
       },
     };
-    this._globalWatcher = new Watcher('deep', () => this._dispatchEvent('listChanged', {}));
```

Comment:
We should double check that emitting `testFilesChanged` event with an empty list of test files is properly ignored in the VSCode extension.

Reply:
will do!